specs/commandline/verify.md

```diff
@@ -37,6 +37,7 @@ Usage:
 
 Flags:
   -h,  --help                    help for verify
+  -o,  --output string           output format, options: 'json', 'text' (default "text")
```

Comment:
@yizha1 @FeynmanZhou This PR is adding a new flag called `--output` into the `notation verify` command. Just want to make sure we are on the same page. And does the flag description look good to you?
And just an open discussion, since this flag is asking the user to type `-o json`, an alternative is to make it a Boolean flag with name `--output-json`. An auto-completion would be feasible here and save some user's typing. What do you think?

Reply:
To add some context, decision to use `--output json` came out of [this](https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw?view#Meeting-Notes) meeting. Also,
* Using `--output` flag provides us flexibility to extend it to yaml or table in future. 
* [AWS](https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-output-format.html), [Azure](https://learn.microsoft.com/en-us/cli/azure/format-output-azure-cli),  [Oracle](https://blogs.oracle.com/cloud-infrastructure/post/exploring-the-search-and-query-features-of-oracle-cloud-infrastructure-command-line-interface), [k8](https://kubernetes.io/docs/reference/kubectl/#formatting-output) all uses `--output` flag, which mean most users already familiar with them.

Reply:
> To add some context, decision to use `--output json` came out of [this](https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw?view#Meeting-Notes) meeting. Also,
> 
> * Using `--output` flag provides us flexibility to extend it to yaml or table in future.
> * [AWS](https://docs.aws.amazon.com/cli/latest/userguide/cli-usage-output-format.html), [Azure](https://learn.microsoft.com/en-us/cli/azure/format-output-azure-cli),  [Oracle](https://blogs.oracle.com/cloud-infrastructure/post/exploring-the-search-and-query-features-of-oracle-cloud-infrastructure-command-line-interface), [k8](https://kubernetes.io/docs/reference/kubectl/#formatting-output) all uses `--output` flag, which mean most users already familiar with them.

I see. Yeah, I agree, `--output` provides more extensibility here.